examples/docs_beta_snippets/setup.py

```diff
@@ -14,7 +14,7 @@
         "Operating System :: OS Independent",
     ],
     packages=find_packages(exclude=["test"]),
-    install_requires=["dagster-cloud"],
+    install_requires=[],
```

Comment:
wonder if `tox.ini` can point at another git repo so we can target latest master

Reply:
I do think it's possible, but complicated slightly because it's in a private repo.